bundles/core/src/main/java/com/adobe/cq/commerce/core/cacheinvalidation/spi/DispatcherCacheInvalidationStrategy.java

```diff
@@ -14,6 +14,8 @@
 
 package com.adobe.cq.commerce.core.cacheinvalidation.spi;
 
+import java.util.List;
+
 import org.osgi.annotation.versioning.ConsumerType;
 
 /**
```

Comment:
Please extend a bit the javadoc with what is the intent of this interface and what customers can achieve with it.